server/database/data/exchangeMoney.js

```diff
@@ -16,6 +16,62 @@ module.exports = async () => {
       agency_price_buy: 696,
       agency_saved_money: 1,
     },
+    {
+      amount: 1250,
+      from: 'ILS',
+      to: 'USD',
+      app_price_sell: 357,
+      app_price_buy: 356,
+      app_saved_money: 4,
```

Comment:
```suggestion
      app_saved_money: 5,                // if we considered that screen price the same with bank price and saved mpney according this will be the difference between both 
```